src/ledger.ts

```diff
@@ -1656,13 +1698,25 @@ export class LedgerV2SignMultisigTransaction extends LedgerBitcoinV2WithRegistra
     return this.signatures.map((sig) => Buffer.from(sig[2]).toString("hex"));
   }
 
+  get SIGNED_PSTBT() {
+    return addSignaturesToPSBT(
+      this.network,
+      this.unsignedPsbt,
+      // array of pubkeys as buffers
+      this.signatures.map((sig) => Buffer.from(sig[1])),
+      // array of sigs as buffers
+      this.signatures.map((sig) => Buffer.from(sig[2]))
+    );
+  }
+
```

Comment:
@Shadouts so this is how I resolved adding signatures for now (the interactions do need them and the "normal" signing flow wouldn't work without something like this). namely I save the psbt that comes in through the constructor on a `this.unsignedPsbt`. that stays raw and is assumed to v0 since this function is from bitcoinjs-lib and v0 only. Later we can change this to just add the sig directly onto `this.psbt` and run `serialize` on that.